### DISCUSSION TITLE
HELP: TypeError: 'module' object is not callable

### DISCUSSION THREAD

**Discussion by JohnSmith2007** [Monika-After-Story/MonikaModDev#8390] [QA - tech support + general] [answered]
My game crashed, and its now showing me this error whenever i try to launch the game. What the hell?! 

[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7682438/traceback.txt)

**Comment by JohnSmith2007** [comment]
> can you reupload the file? It doesn't seem to download properly

Sure. 
[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7682447/traceback.txt)

**Comment by multimokia** [comment]
Hmm, looking at this, it's before loading script. Since `early` (the earliest phase we start loading stuff) doesn't throw errors, the next phase is `init`, which would report different text.

This leads me to believe there's some issues in the actual renpy files. Have you tried reinstalling DDLC and MAS?

**Comment by JohnSmith2007** [comment]
tried opening it again, and now its giving me this.
[errors.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7682460/errors.txt)
![Screenshot (64)](https://user-images.githubusercontent.com/45465767/145339609-c4d25945-20fd-4148-aead-002d22e1bbe1.png)

**Comment by multimokia** [comment, marked as answer]
Very odd. We haven't touched zz_calendar for around 6 months now. Have seen this error before but it's odd. Definitely give a reinstall a shot. You won't lose your progress.

**Reply by mahaski** [reply]
Excuse me, i have a similiar (most likely the same) problem, and i see that the Author have their problem solved, can you please elaborate on how to reinstall it? im not very knowledgable in this topic and i dont understand on how reinstalling the game and mod as a whole wouldnt lose the game progress.

I also would like to apologize in advance for my rudeness as this is my first time posting anything in this site.
![Screenshot (2264)](https://github.com/Monika-After-Story/MonikaModDev/assets/135545875/09a7cb9b-2caa-4084-8b40-8e294ca8e5b3)

**Reply by Booplicate** [reply]
You're using a bogus version of MAS, make a new [DDLC install](https://ddlc.moe/) and then [install MAS](https://monikaafterstory.com/releases.html). Just follow the installer instructions, you won't lose your progress.

**Reply by mahaski** [reply]
Thank goodness it's solved now, thank you so much! it turns out i did a few missstep while reinstalling, but now everything's back to normal.

**Comment by JohnSmith2007** [comment]
Ok then. are there any files i should transfer from the previous install to the new one, or do i just do a fresh install?

**Comment by JohnSmith2007** [comment]
Thank fuck, it worked. Thank you so much

**Comment by multimokia** [comment]
No problem!
